backend/src/core/middlewares/api-key.middleware.ts

```diff
@@ -0,0 +1,35 @@
+import { loggerWithLabel } from '@core/logger'
+import { Request, Response } from 'express'
+import { ApiKeyService } from '@core/services'
+
+const logger = loggerWithLabel(module)
+
+const deleteApiKey = async (
+  req: Request,
+  res: Response
+): Promise<Response | void> => {
+  const { apiKeyId } = req.params
+  const userId = req.session?.user?.id
+
+  const deletedRows = await ApiKeyService.deleteApiKey(
+    userId.toString(),
+    +apiKeyId
+  )
+  if (deletedRows <= 0) {
```

Comment:
**[nit]** same as previous nit, if it cannot be `< 0`, why don't we just do `deletedRows === 0`

Reply:
updated